<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Music_Playlist_choice</class>
 <widget class="QDialog" name="Music_Playlist_choice">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>357</width>
    <height>280</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>280</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>19</y>
     <width>341</width>
     <height>241</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
    </font>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>120</y>
       <width>341</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Nirmala UI</family>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">  #pushButton_2{  border-width: 9px;
    border-radius: 20px;
    border-color:rgba(78, 12, 12, 0.1); 
	background-color : rgba(96, 96, 55, 0.1);

    font: bold 14px;
    padding: 1px}
#pushButton_2:hover{
background-color:	rgba(116, 114, 157, 0.2);
}
</string>
     </property>
     <property name="text">
      <string>         Add all music in list</string>
     </property>
     <property name="icon">
      <iconset resource="qml.qrc">
       <normaloff>:/Image/Add.png</normaloff>:/Image/Add.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>341</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Nirmala UI</family>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">  #pushButton{  border-width: 9px;
    border-radius: 20px;
    border-color:rgba(78, 12, 12, 0.3); 
	background-color : rgba(96, 96, 55, 0.1)

;
    font: bold 14px;
    padding: 1px}
#pushButton:hover{
background-color:	rgba(116, 114, 157, 0.2);
}
</string>
     </property>
     <property name="text">
      <string>Add music has been choice</string>
     </property>
     <property name="icon">
      <iconset resource="qml.qrc">
       <normaloff>:/Image/Add.png</normaloff>:/Image/Add.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>-5</y>
       <width>341</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Nirmala UI</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Playlist list</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QQuickWidget" name="quickWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>341</width>
       <height>200</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>341</width>
       <height>200</height>
      </size>
     </property>
     <property name="resizeMode">
      <enum>QQuickWidget::SizeRootObjectToView</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>220</y>
       <width>80</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBar">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>220</y>
       <width>251</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-width: 9px;
    border-radius: 20px;
    border-color:rgba(78, 12, 12, 0.1); 
	background-color : rgba(96, 96, 55, 0.1);</string>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QQuickWidget</class>
   <extends>QWidget</extends>
   <header location="global">QtQuickWidgets/QQuickWidget</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="qml.qrc"/>
 </resources>
 <connections/>
</ui>
